include/kokkos-utils/tests/fixtures/ExecutionSpaceInstanceFixture.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef KOKKOS_UTILS_TESTS_HELPERS_HPP
+#define KOKKOS_UTILS_TESTS_HELPERS_HPP
+
+#include "kokkos-utils/concepts/ExecutionSpace.hpp"
+
+namespace Kokkos::utils::tests
+{
+
+//! Fixture class that creates an new execution space instance.
+template <Kokkos::utils::concepts::ExecutionSpace Exec>
+class ExecutionSpaceInstanceFixture : public virtual ::testing::Test
+{
+public:
+    ExecutionSpaceInstanceFixture() : exec(Kokkos::Experimental::partition_space(Exec{}, 1)[0]) {}
+
+protected:
+    Exec exec {};
```

Comment:
```suggestion
    Exec exec;
```
(if we'd want to signal that it's the default constructor that is constructing this member)

Reply:
:heavy_check_mark: